src/screens/profile-screen/ProfileScreen.tsx

```diff
@@ -1,77 +1,63 @@
-import { NativeStackScreenProps } from '@react-navigation/native-stack'
+import { makeGetProfile } from 'audius-client/src/pages/profile-page/store/selectors'
```

Comment:
I think we should move things to common incrementally as we build out features, that way we don't have to go back  to find and update all the dependencies outside of common

Reply:
sounds great